.setup/bin/setup_sample_courses.py

```diff
@@ -61,6 +61,9 @@ def main():
     """
     global DB_ONLY
 
+    # To make Iris testing possible, need to seed random
+    random.seed(10090542)
```

Comment:
I agree about setting the seed on a course-by-course basis (and again for usernames). I'll get that added in.

I don't want to use the course name because using non-numeric types in `seed()` causes them to be passed to `hash()` which is OS-dependent and may not be deterministic.